packages/data-stores/jest.config.js

```diff
@@ -1,3 +1,10 @@
+const jestPreset = require( '../../test/packages/jest-preset.js' );
+
 module.exports = {
-	preset: '../../test/packages/jest-preset.js',
+	...jestPreset,
+	testPathIgnorePatterns: [
+		...( jestPreset.testPathIgnorePatterns || [] ), // Spread existing ignore patterns (if any)
+		// Ignore test utils files
+		'[a-zA-Z/]*test/utils.ts[x]?',
+	],
```

Comment:
The change in this file is required for the utils file to live inside the test folder and jest not take it as a test file.

Reply:
Based on `test/packages/jest.config.js` implementation, seems like we can have our own `jest.config.js` (inside reader folder) file with our specific implementation.